app/grandchallenge/documentation/admin.py

```diff
@@ -11,6 +11,7 @@ class Meta:
         model = DocPage
         fields = (
             "title",
+            "slug",

```

Comment:
I think this does not work because `AutoSlugField` is not editable. To still show it, you could add it to `readonly_fields` in the `DocPageAdmin`. Alternatively, we might need to change the slug field in the DocPage model, I'm afraid.